### ISSUE TITLE
[wg/webmachinelearning] Web Machine Learning Working Group Charter

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/strategy#488]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/machine-learning-charter/charter.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/machine-learning-charter/charter.html)

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2023%2F04%2Fweb-machine-learning-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fmachine-learning-charter%2Fcharter.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/webmachinelearning)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [X] Existing WG recharter 

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Communities suggested for outreach

Browser vendors, AI systems developers, ML processing units vendors, ML framework developers

## Known or potential areas of concern

N/A

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

https://github.com/w3c/machine-learning-charter/issues

## Anything else we should think about as we review? 

The sister Web Machine Learning Community Group recently adopted a set of [LLM-based browser APIs for incubation](https://webmachinelearning.github.io/incubations/) - the [Working Group is not ready to adopt them as tentative deliverables](https://github.com/w3c/machine-learning-charter/issues/36) yet, but is signalling its future interest in doing so to call for the attention of the community on this space.

Note: proposed chairs should be copied @... on this issue.
cc @anssiko

**Comment by himorin** [comment]
charter history table might need to have a link to current running charter (and also additional row of this proposed charter).

**Comment by dontcallmedom** [comment]
history table updated in https://github.com/w3c/machine-learning-charter/commit/133f23a29f808fcb3fb554b61e980526cd836301

**Comment by plehegar** [comment]
[Advance notice](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JanMar/0007.html)

**Comment by himorin** [comment]
No comment or request from i18n

**Comment by ruoxiran** [comment]
APA is OK for this Charter.  
one thing we hope to get your awareness of is, that APA/RQTF had an ongoing AI-accessibility NOTE, which is currently in ED at: https://github.com/w3c/ai-accessibility

**Comment by svgeesus** [comment]
Overall the charter looks good. 

One minor question: Why has the word "specifications" been replaced by "Recommendations"?  The template used to say that, and we changed it once CR Snapshots also gained patent policy protection.

> To promote the widest adoption of Web standards, W3C seeks to issue Recommendations that can be implemented, according to this policy, on a Royalty-Free basis. 

In other words is that an intentional change, or just a cop-paste error from an earlier charter or earlier template?

**Comment by dontcallmedom** [comment]
thanks for spotting this @svgeesus ! this was something I miss when doing the charter template alignment, now fixed

**Comment by caribouW3** [comment]
The timeline seems slightly too optimistic, unless you are planning to publish a (final) CR snapshot right away?
By the time the AC review is over, the group would already be in PR, then it raises another question: since no other deliverable is listed, why rechartering for 2 years if the spec is nearly done?  It seems that a simple extension would have been sufficient.

**Comment by dontcallmedom** [comment]
@caribouW3 thanks for flagging this, I had missed updating the timeline; I've raised https://github.com/w3c/machine-learning-charter/issues/43 to track fixing this - for sake of clarity, it's entirely clear we won't finish WebNN by the end of this quarter; the charter will be updated with a more up-to-date estimate.

**Comment by dontcallmedom** [comment]
@caribouW3 charter now updated with a new completion timeline

**Comment by caribouW3** [comment]
+1  (you should update in section 3.3 as well)

**Comment by plehegar** [comment]
very minor consideration in the motivation section:
s/, and Speech Recognition and Speech Synthesis/, and Speech Recognition and Synthesis/

or
s/, and Speech Recognition and Speech Synthesis /, Speech Recognition, and Speech Synthesis/

**Comment by dontcallmedom** [comment]
I'm not a native English speaker, but I think the current capitalization and punctuation correctly reflect the usage of "Speech Recognition" and "Speech Synthesis" as two separate types of processing, both of which are the subject of "enable".

**Comment by plehegar** [comment]
As part of its scope, the group has a "priority on building blocks required by well-known model architectures".

What progress, if any, has been made around https://github.com/webmachinelearning/webnn/issues/350 ?

**Comment by dontcallmedom** [comment]
Splitting into multiple sentences make sense for readability, indeed - done

**Comment by plehegar** [comment]
No comment from Privacy WG

**Comment by plehegar** [comment]
[VOTE by 2025-03-05/06: Proposed Charter for the Web Machine Learning Working Group](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JanMar/0029.html)

**Comment by plehegar** [comment]
[Announced](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JanMar/0058.html)
